client/src/components/AuthHeader/AuthHeader.tsx

```diff
@@ -1,27 +1,111 @@
+  Menu,
+  MenuItem,
+} from '@material-ui/core';
+import MenuIcon from '@mui/icons-material/Menu';
+import logo from '../../Images/logo.png';
 
 interface Props {
-  linkTo: string;
   asideText: string;
-  btnText: string;
+  loginButtonLinkTo: string;
+  signupButtonLinkTo: string;
+  loginButtonText: string;
+  signupButtonText: string;
 }
 
-const AuthHeader = ({ linkTo, asideText, btnText }: Props): JSX.Element => {
+const AuthHeader = ({
+  asideText,
+  loginButtonLinkTo,
+  signupButtonLinkTo,
+  loginButtonText,
+  signupButtonText,
+}: Props): JSX.Element => {
   const classes = useStyles();
+  const theme = useTheme();
+  const isSmallScreen = useMediaQuery(theme.breakpoints.down('sm'));
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+  const menuIsOpen = Boolean(anchorEl);
+  const handleMenuClick = (event: MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+  const handleMenuClose = () => {
+    setAnchorEl(null);
+  };
 
   return (
-    <Box p={1} className={classes.authHeader}>
-      <Typography className={classes.accAside}>{asideText}</Typography>
-      <Link to={linkTo} className={classes.link}>
-        <Button color="inherit" className={classes.accBtn} variant="contained">
-          {btnText}
-        </Button>
-      </Link>
+    <Box className={classes.authHeader}>
+      <AppBar position="static" className={classes.authAppBar}>
+        <Toolbar className={classes.authToolBar}>
+          <Box>
+            <img src={logo} alt="Loving Sitter Logo" />
+          </Box>
+          {isSmallScreen ? (
+            <>
+              <Box className={classes.authLinks}>
+                <IconButton
+                  id="menu-button"
+                  aria-controls="menu-button"
+                  aria-haspopup="true"
+                  aria-expanded={menuIsOpen ? 'true' : undefined}
+                  onClick={handleMenuClick}
+                  size="medium"
+                  edge="start"
+                  color="inherit"
+                  aria-label="menu"
+                >
+                  <MenuIcon htmlColor={theme.palette.primary.main} />
+                </IconButton>
+              </Box>
+              <Menu
+                id="menu-button"
+                anchorEl={anchorEl}
+                open={menuIsOpen}
+                onClose={handleMenuClose}
+                MenuListProps={{
+                  'aria-labelledby': 'basic-button',
+                }}
+              >
+                <MenuItem onClick={handleMenuClose}>{loginButtonText}</MenuItem>
+                <MenuItem onClick={handleMenuClose}>{signupButtonText}</MenuItem>
+              </Menu>
+            </>
+          ) : (
+            <Box className={classes.authLinks}>
+              <Typography className={classes.accAside}>{asideText}</Typography>
+              <Link to={loginButtonLinkTo} className={classes.link}>
+                <Button color="primary" variant="outlined">
```

Comment:
```suggestion
                <Button component={Link} to={loginButtonLinkTo} color="primary" variant="outlined">
```
The MUI Button and react-router's Link can be combined :)
Ditto throughout!

Reply:
Resolved by a4912e06cc41cb7e511c0ab8b8d2c0fe5077f483